ssg/build_remediations.py

```diff
@@ -146,6 +151,81 @@ def get_populate_replacement(remediation_type, text):
     sys.exit(1)
 
 
+def from_file(file_path, env_yaml):
+    mod_file = []
+    config = defaultdict(lambda: None)
```

Comment:
Here, we make sure that we will end up with dictionary containing all keys, but some of them may be `None`. Later in the code, you have to check whether the associated value is not `None`, so you have a sequence of `if`s instead of one loop (in `write_fixes`). Check it out!